pr_tester.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+############################################################
+#
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+############################################################
+
+__author__="${FULL_NAME}"
+__copyright__="Copyright (c) 2017 OSIsoft, LLC"
+__license__="Apache 2.0"
+__version__="${VERSION}"
+
+
+# DB setup
+jenkins_db_user='jenkins'
+foglamp_db_user='foglamp'
+postgres_db_user='postgres'
+
+PGPASSWORD=${postgres_db_user} psql -U ${postgres_db_user} -h localhost -f src/sql/foglamp_ddl.sql ${postgres_db_user}
+PGPASSWORD=${foglamp_db_user} psql -U ${foglamp_db_user} -h localhost -f src/sql/foglamp_init_data.sql ${foglamp_db_user}
+PGPASSWORD=${foglamp_db_user} psql -c "GRANT ALL ON DATABASE ${foglamp_db_user} to ${jenkins_db_user};" -U ${foglamp_db_user} -h localhost
```

Comment:
why is the jenkins users of concern here?

Reply:
This was needed before merge https://scaledb.atlassian.net/browse/FOGL-506 into develop. These changes are stale now as per latest develop, will be fix in another round commit.